frontend/src/core/components/Help/Help.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { Button } from "react-bootstrap";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faQuestionCircle } from "@fortawesome/free-solid-svg-icons";
+
+import "./Help.scss";
+
+const Help = () => {
+  return (
+    <div className={"help"}>
+      <Button variant={"link"}>
+        <FontAwesomeIcon icon={faQuestionCircle} />
+      </Button>
+    </div>
+  );
+};
+
+export default Help;
```

Comment:
For component, we need to follow class based component style.
Please use _core/components/AuthProviderButton.js_ as example.

Also use propTypes property of component if is needed.